src/services/asset-discovery-service.ts

```diff
@@ -9,17 +10,19 @@ interface AssetDiscoveryOptions {
   networkIdleTimeout?: number
 }
 
+const DEFAULT_PAGE_POOL_SIZE = process.env.PERCY_POOL_SIZE
```

Comment:
we'll shift this to use the `.percy.yml` config file later. Happy to leave this is for now, but don't rely on this